### PR TITLE
Django 4.2 fixes

### PR DESCRIPTION
**Pull request by Pablo-Balaudo**
Update the code to be `Python 3` compatible.
Update the code to be `Django 4.2` compatible.
Updated documentation.
Applied `black` and `pylint`.

### CODE REVIEW

File: zebra/signals.py [hunk 1]
```diff
@@ -53,12 +53,12 @@
 # v1 API webhooks
 WEBHOOK_ARGS = ["customer", "full_json"]
```
Comment (ganiserb) [inline zebra/signals.py:53]: How does this get translated? Will the webhooks receive whatever this is?

**Comment by gbourdin** [comment]
Don't forget to update the usage example in the README to reflect the new way to add url patterns since this now only supports Django 4 and `django.conf.urls.url()` was deprecated

**Comment by nmallare** [comment]
> Don't forget to update the usage example in the README to reflect the new way to add url patterns since this now only supports Django 4 and `django.conf.urls.url()` was deprecated

Old friend alert!

File: zebra/signals.py (hunk 1 above)
Comment (Pablo-Balaudo) [reply zebra/signals.py:53]: Allegedly yes. The `providing_args` that I removed were used only for documentation purposes.

**Review by ganiserb — APPROVED** [review]

**Pull request merged**